config/secrets.yml.dev

```diff
@@ -78,15 +78,15 @@ dev_settings: &DEV_SETTINGS
   custom_indices:
     elasticsearch:
       reader:
-        host: http://localhost:9200
+        hosts:
+          - http://localhost:9200
         user: elastic
         password: changeme
-        log: false
       writers:
-        - host: http://localhost:9200
+        - hosts:
```

Comment:
does this need to be a list since we only have one for writters?

Reply:
There is only one host in development & test environments, but in production there will be an array of hosts.

Reply:
correct but you are making - hosts: a list for writers.  - http://localhost:92000 is a list for the hosts.

Reply:
We currently only using a single writer in all environments, but I will most likely add another writer when we upgrade to ES 6. Each writer will have an array of hosts. That will (theoretically) allow us to temporarily write to two clusters, enabling us to rebuild our indices in ES 6 while keeping data in sync between ES 5 and ES 6.